shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -44,6 +46,8 @@ - (void)await:(shell::VsyncWaiter::Callback)callback {
 }
 
 - (void)onDisplayLink:(CADisplayLink*)link {
+  _traceCounter = !_traceCounter;
+  TRACE_COUNTER1("flutter", "OnDisplayLink", _traceCounter);
```

Comment:
I'm not sure what you're trying to do here.  Do you want an instant trace event?